templates/checkout/summary.html

```diff
@@ -7,6 +7,7 @@
     {% csrf_token %}
     <h2>{% trans "Billing address" context "Checkout summary title" %}</h2>
     {% include "checkout/snippets/addresses_form.html" with addresses_form=addresses_form addresses=additional_addresses address_form=address_form only %}
+    <p><a href="{% url 'checkout:shipping-method' %}">{% trans "Change shipping method" context "Checkout shipping method" %}</a></p>
```

Comment:
Is this change related to this PR?

Reply:
@maarcingebala Yes It is. Issue #2234 also including ability for returning from summary page to shiping-method page, so I added this simple link. It looks as you can see on screenshot. Also I corrected button spacing on mobile view.
<img width="415" alt="cart1" src="https://user-images.githubusercontent.com/7680353/43462304-0b48670e-94d6-11e8-8f24-3da6a1ab1f13.png">
<img width="861" alt="cart2" src="https://user-images.githubusercontent.com/7680353/43462305-0b709666-94d6-11e8-85e3-6c4963484951.png">